geode-cq/src/distributedTest/java/org/apache/geode/cache/query/cq/dunit/CqPerfDUnitTest.java

```diff
@@ -57,15 +58,16 @@
  */
 @Category({ClientSubscriptionTest.class})
 public class CqPerfDUnitTest extends JUnit4CacheTestCase {
-
+  private final Logger log = LogService.getLogger();
+  @SuppressWarnings("CanBeFinal")
```

Comment:
Is there a reason not to make the field private?

Reply:
done